src/components/App/App.jsx

```diff
@@ -1,35 +1,23 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import { Route, Routes } from "react-router-dom";
+import LandingPage from "../LandingPage/LandingPage";
+import SearchPage from "../SearchPage/SearchPage";
+import AboutPage from "../About/About";
+// import Preloader from "../Preloader/Preloader";
+import "./App.css";
 
 function App() {
-  const [count, setCount] = useState(0)
-
   return (
     <>
```

Comment:
`<>` empty tags are not needed if you have a common tag that wraps the whole markup of the component